packages/widget-react/src/components/form/ChainOptions.tsx

```
@@ -4,29 +4,33 @@ import WidgetTooltip from "../WidgetTooltip"
       <div className={styles.grid}>
         {chains.map(({ chainId, name, logoUrl }) => (
-          <WidgetTooltip label={name} key={chainId} disableHoverableContent>
-            <button
-              type="button"
-              className={clsx(styles.item, { [styles.active]: chainId === value })}
-              onClick={() => onSelect(chainId)}
-            >
-              <Image src={logoUrl} width={28} height={28} circle />
-            </button>
-          </WidgetTooltip>
+          <IndicatorBadge hidden={!getShowIndicator?.(chainId)} className={styles.item}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)